components/script/dom/validation.rs

```diff
@@ -1,5 +1,10 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+use dom::element::Element;
+use dom::validitystate::ValidityStatus;
 
-pub trait Validatable {}
+pub trait Validatable {
+    fn as_element_validatable(&self) -> &Element;
```

Comment:
Method as_maybe_validatable is defined in element.rs file which returns an instance of Validatable trait. Attaching the screenshot for the same. Please verify once.

![as_maybe_validatable](https://cloud.githubusercontent.com/assets/21151022/19947234/9446e250-a11d-11e6-9134-556e74009b34.PNG)